src/xhr.js

```diff
@@ -43,6 +52,9 @@ class Xhr {
      * @private @type {function(string, ?FetchInit=):!Promise<!FetchResponse>}
      */
     this.fetch_ = win.fetch || fetchPolyfill;
+    // We want to call `fetch_` unbound from any context since it could
+    // be either the native fetch or our polyfill.
+    this.fetch_ = this.fetch_.bind(null);
```

Comment:
please make the `this.fetch_` above `@const` and combine these two lines. E.g. `this.fetch_ = (a || b).bind(null)`